addons/account/models/company.py

```diff
@@ -202,6 +206,9 @@ def _get_company_root_delegated_field_names(self):
             'tax_exigibility',
         ]
 
+    def _compute_show_taxes(self):
+        pass
```

Comment:
would it not be better to set the boolean value to True by default on all records? It can then be set to False on overridden functions, but at least this way, you set a default value

Reply:
This is a computed stored column. So, If I set default here, It will never call the overwrite function.
Or, Am I understanding it wrong?

Reply:
but I see the issue that that the column wont be shown up if there is no already installed us or br pack

Reply:
Done